tasks/build.yml

```diff
@@ -0,0 +1,51 @@
+---
+# This task file assumes that {{ item.key }} is the name and {{ item.value }}
```

Comment:
For future reference: You can change the variable name of the looped item using [loop_control](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#loop-control)